book/src/developers/contributing/build_instructions/windows.md

```diff
@@ -9,12 +9,6 @@ If you don't already have Rust install it
 $ winget install Rustlang.Rustup
 ```
 
-Install clang/llvm as it is required to compile rocksdb
-If you don't already have Rust install it
-```sh
-$ winget install LLVM.LLVM
-```
-
```

Comment:
If we are removing this in our build instructions why do we still install it in our CI?

Reply:
Good catch, reverted this because LLVM is required to compile c-kzg.